modules/tile-converter/src/i3s-converter/i3s-converter.ts

```diff
@@ -218,13 +218,13 @@ export default class I3SConverter {
         loadOptions: {
           _nodeWorkers: true,
           reuseWorkers: true,
-          basis: {format: 'rgba32'}
-          // TODO - should no longer be needed with new Node workers
-          // 'basis-nodejs': {
-          //   format: 'rgba32',
-          //   workerUrl: './modules/textures/dist/basis-worker-node.js'
-          // },
-          // 'draco-nodejs': {workerUrl: './modules/draco/dist/draco-worker-node.js'}
+          basis: {
+            format: 'rgba32',
+            // We need to load local fs workers because nodejs can't load workers from the Internet
+            workerUrl: './modules/textures/dist/basis-worker-node.js'
+          },
+          // We need to load local fs workers because nodejs can't load workers from the Internet
```

Comment:
We could generate local URLs in worker-utils.

If we load the URL first then we can create the worker from source text?

Reply:
This path should be same as created by `install-dependencies` script. I've made it this way to work in test environment but it is arbitrary by design.